src/Microsoft.SqlTools.ServiceLayer/Localization/sr.cs

```diff
@@ -3061,1525 +3061,8976 @@ public static string SqlBindingsNet5NotSupported
             }
         }
 
-        public static string ConnectionServiceListDbErrorNotConnected(string uri)
+        public static string Statement

```

Comment:
there are a ton of string updates here.  I assume you verified we need all these.